products/oneplus.md

```diff
@@ -0,0 +1,66 @@
+---
+title: OnePlus
```

Comment:
Not very happy with the title, but I don't like "OnePlus Phone" either. Anyone has suggestions?

Reply:
I already see "Fairphone", "Google Pixel", "Samsung Mobile" (🤮)...
So looks fine to me like this.
Or eventually "OnePlus phones"

Reply:
@captn3m0 do we leave OnePlus ? Or can we settle on something else ? so that we can merge this

Reply:
Let's leave it at OnePlus for now.